mlops_utilities/actions.py

```diff
@@ -94,36 +100,33 @@ def run_pipeline(
     :param dryrun: should be run in test mode without real execution. If true then the method returns only arguments
     :param pipeline_params: additional parameters for pipeline
     """
-    sagemaker_client = boto3.client('sagemaker')  # Can not be cut off because it could not be presented as string
+    sagemaker_client = boto3.client(
+        "sagemaker"
+    )  # Can not be cut off because it could not be presented as string
     now = datetime.today()
     now_str = helpers.get_datetime_str(now)
-    pipe_exec_name = f'{execution_name_prefix}-{now_str}'
+    pipe_exec_name = f"{execution_name_prefix}-{now_str}"
     start_pipe_args = {
-        'PipelineName': pipeline_name,
-        'PipelineExecutionDisplayName': pipe_exec_name,
-        'PipelineParameters': [
-            {
-                'Name': k,
-                'Value': str(v)
-            }
-            for k, v in pipeline_params.items()
-        ],
-        'ClientRequestToken': helpers.ensure_min_length(pipe_exec_name, 32)
```

Comment:
Why did we remove client token here? We are refactoring, not changing the previously implemented behavior.

Reply:
This is not resolved yet